pkg/reconciler/eventpolicy/controller_test.go

```diff
@@ -31,7 +35,12 @@ import (
 func TestNew(t *testing.T) {
 	ctx, _ := SetupFakeContext(t)
 
-	c := NewController(ctx, configmap.NewStaticWatcher())
+	c := NewController(ctx, configmap.NewStaticWatcher(&corev1.ConfigMap{
```

Comment:
Question: Do we need to add some tests to validate CM changes are getting reconciled in the `EventPolicy` reconciler?

Reply:
I think we should be fine here, as we have this similar in most of the reconcilers and should see this in the e2e test later too